src/gatekeeper/Dockerfile

```diff
@@ -9,7 +9,7 @@
 #     IBM Corporation - initial API and implementation
 ################################################################################
 
-FROM node:12-alpine
+FROM ppc64le/node:12-alpine
```

Comment:
This is changing the main dockerfile that would affect all platforms

Reply:
The fix is probably to rename this file `Dockerfile_x86_64` and create a new `Dockerfile_ppc64le` with this change in.
The `src/gatekeeper/build` script already gets passed an argument with the dockerfile name:
https://github.com/eclipse/codewind/blob/6a7590a49dd5979599a46c271c7b811557e8d1d7/script/build.sh#L77
it's just ignoring it. It should use it as the --file argument like pfe:
https://github.com/eclipse/codewind/blob/ae2e9051f23e3b0270c2cacbc0966e7da9cb173e/src/pfe/build#L33

(The same should probably be done for the keycloak container.)